tests/virt_autotest/virsh_virtual_network.pm

```diff
@@ -0,0 +1,352 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Managing Virtual Networks with virsh included
+# the following priorities 4 types of virtual network:
+# P1 HOST bridge network 
+# P2 NAT based network
+# P3 Routed network
+# P4 Isolated network
+# Maintainer: Leon Guo <xguo@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use set_config_as_glue;
+use version_utils 'is_sle';
+
+
+our $virt_host_bridge = 'br0';
+sub prepare_network {
+
+     my $config_path  = "/etc/sysconfig/network/ifcfg-br0";
```

Comment:
ifcfg-'br0': a fixed name is used here, is it something to do with the default $virt_host_bridge = 'br0'? or they are totally different thing?

Reply:
Both $virt_host_bridge = 'br0' and file ifcfg-'br0' were related to each other here. 
If there was created host bridge as br0 as default on the testing machines, also, there will be create the ifcfg-'br0'  file tool. if not, there will be create host bridge as br0 on the testing machine.

Reply:
Code changed as below:
-     my $config_path  = "/etc/sysconfig/network/ifcfg-br0";
+    my $config_path = "/etc/sysconfig/network/ifcfg-$virt_host_bridge";